temp/automatic_ticket_revocations_proposal.md

```diff
@@ -0,0 +1,131 @@
+# Proposal Structure
+
+The structure of this proposal follows the pattern established by previously proposed consensus changes.  There are two main sections of this proposal:
+
+- General Audience
+- Technical Details
+
+The general audience section provides a high-level overview of the proposal.  It includes details suitable for all stakeholders, including those unfamiliar with the inner workings of consensus.
+
+The technical details section contains a more in-depth discussion of the relevant technical aspects, implementation specifics, and rationale for the chosen semantics of the proposal.
+
+# General Audience
+
+## High-Level Overview
+
+This proposal intends to significantly improve the stakeholder user experience around ticket revocations by automatically revoking missed or expired tickets.
```

Comment:
I might suggest the following slight tweak to the verbiage here to make it less ambiguous:

```suggestion
This proposal intends to significantly improve the stakeholder user experience around ticket revocations by removing the requirement for stakeholders to manually revoke missed and expired tickets.
```

I say that because, after thinking about it from the perspective of someone who is not already familiar with everything, which is the intent of this section, merely saying "by automatically revoking" seems a bit ambiguous to me. My first thoughts would be "Who or what is doing that automatic revocation?"  Finally, I would suggest ` and` over `or` since it's inclusive of both.

It's probably already reasonable enough with any changes, so it's obviously up to you if you think it's worth tweaking or not.

Reply:
I think that is a good suggestion, updated.